### PR TITLE
Language fix for MadBricktree in Beatmap Nominators

### PR DESCRIPTION
**Pull request by tailsdk**
He is actually Korean he just can't change his flag and therefor he doesn't actually know Chinese, but instead knows Korean

### CODE REVIEW

**Review by aceticke — APPROVED** [review]
seems good, for the future you can use osu-wiki bin to do the languages for you and push to one commit https://github.com/cl8n/osu-wiki-bin

approval from madbrick would be nice too, as nothing on their profile mentions this

**Comment by tailsdk** [comment]
Sure let me get him in voice with him rn

**Comment by fishinwashingmachine** [comment]
madbricktree here
yea I'm korean

File: wiki/People/Beatmap_Nominators/en.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
 | ::{ flag=KR }:: [Kim_GodSSI](https://osu.ppy.sh/users/2218047) | Korean |
 | ::{ flag=KR }:: [Lami-](https://osu.ppy.sh/users/16505281) | Korean |
 | ::{ flag=PH }:: [lenpai](https://osu.ppy.sh/users/5314573) | Filipino |
-| ::{ flag=CN }:: [MadBricktree](https://osu.ppy.sh/users/4790958) | Chinese |
+| ::{ flag=CN }:: [MadBricktree](https://osu.ppy.sh/users/4790958) | Korean |
```
Comment (peppy) [inline wiki/People/Beatmap_Nominators/en.md:183]: I'm 909% sure this is wrong. (CN pointing to Korean)

**Review by peppy — CHANGES_REQUESTED** [review]

File: wiki/People/Beatmap_Nominators/en.md (hunk 1 above)
Comment (tailsdk) [reply wiki/People/Beatmap_Nominators/en.md:183]: No he is korean he was just in china when he made thw account and he is just stuck on chinese flag now

Comment (Protastic101) [reply wiki/People/Beatmap_Nominators/en.md:183]: https://github.com/ppy/osu-wiki/pull/9116#issuecomment-1500925504 As confirmed by the man himself (you can also see he does a bunch of mods in Korean, and gets very lost and confused when people mod his stuff in Chinese)

**Pull request merged**